s4e-web/src/app/views/map-view/state/product/product.service.ts

```diff
@@ -198,6 +200,7 @@ export class ProductService {
 
         return !!data.sceneId;
       }),
+      logIt('getLastAvailableScene$.setSelectedDate'),
```

Comment:
It's also rather unnecessary log, but there are several such invocations throughout the app, so I'm not 100% sure about this:
![image](https://user-images.githubusercontent.com/583197/108035019-a85acd00-7036-11eb-9500-9a7fa47b8331.png)